ops/upgradetf.sh

```diff
@@ -12,5 +12,5 @@ terraform fmt -recursive
 for d in $TERRAFORM_DIRS
 do
     echo "Initializing/formatting/validating $d";
-    (cd $d && rm .terraform.lock.hcl && terraform init -upgrade -backend=false && terraform validate)
+    (cd $d && rm .terraform.lock.hcl || true && terraform init -upgrade -backend=false && terraform validate)
```

Comment:
is the `|| true` intentional?

Reply:
It was! I found myself in a situation where I didn't have all the lock files for all the environments, so the command failed. I can revert it now, though. I appreciate the call-out.

Reply:
So, I passed the `-f` flag here. This should cover my issues and make is simpler to approach any issues with this script.